### DISCUSSION TITLE
Removing website embeds on shortened links

### DISCUSSION THREAD

**Discussion by StoneMcYT** [shlinkio/shlink#2103] [Help wanted]
### Shlink version

4.1.0

### PHP version

Docker

### How do you serve Shlink

Docker image

### Database engine

SQLite

### Database version

Using the inbuilt one for docker

### Summary

Playing around with shlink for the first time and I was wondering if its possible to disable webpage embeds? Because on some sites eg discord shows an embed. Was wondering if there was a way to disable embeds so that I dont have to manually close them everytime and for certain apps that cant close embeds.

![image](https://github.com/shlinkio/shlink/assets/67421882/a12fcae2-875a-4aec-bb5a-94cf5c02caee)

Thanks in advance!

**Comment by acelaya** [comment]
Did you make the short URLs crawlable via robots.txt?

![Screenshot_20240425-034304.png](https://github.com/shlinkio/shlink/assets/2719332/7067a44d-67a3-4f64-bcb7-f3d623888e52)

If you did, does not making them crawlable help?

**Comment by acelaya** [comment]
Closing, as there's no follow-up from reporter.
